### DISCUSSION TITLE
在追番追剧选项卡屏蔽PV和预告片

### DISCUSSION THREAD

**Discussion by Summer-Tail** [the1812/Bilibili-Evolved#2006] [功能建议 / Ideas]
rt
每次去翻全是预告或者pv烦死

**Comment by Summer-Tail** [comment]
是在受不了了,自己弄了个油猴脚本
https://greasyfork.org/zh-CN/scripts/428781-bilibili%E8%BF%BD%E7%95%AA%E8%BF%BD%E5%89%A7-%E7%A7%BB%E9%99%A4%E9%A2%84%E5%91%8Apv

**Comment by the1812** [comment]
我可以改善下动态过滤器, 届时加几个屏蔽关键词就行

**Reply by Summer-Tail** [reply]
大佬,我这几行代码写的稀烂,而且我还研究了好几个小时,如果你能早点回复我多好T.T

**Reply by the1812** [reply]
能自己动手是好事(
而且等我更新可没这么快
